### ISSUE TITLE
Add poll rates for 15/30 seconds - feature request

### ISSUE THREAD

**Issue by ukmerlin** [jaszhix/NoMansConnect#127]
Because of the way screenshots work my work flow is to land somewhere especially space stations and quickly maneuver so that I can see the back end of the station in wide view (JJ's Station Ventilation mod)  :) 
Then I have to wait there for at least a minute to ensure that NMC has taken a screen shot of the landing.
Would be useful if poll rate could be brought down quicker, say at least 30 secs.

**Comment by KaneHart** [comment]
I actually wonder instead of increase poll rates there could be some sort of streamed tech added instead. More like a live feed and make it kind of more enjoyable to follow?

I kind of like to see things a bit more streamlines in a sense if that makes sense a smoother look to it if possible.

**Comment by jaszhix** [comment]
Yeah, that's something I've considered previously. The API is built with the Django web framework, which does support web sockets with an officially supported plugin. Worth looking into, but in the mean time I can increase the polling rate options.
